src/app/clusters/mode-select-server/mode-select-server.cpp

```diff
@@ -134,6 +135,29 @@ void emberAfModeSelectClusterServerInitCallback(EndpointId endpointId)
             // Initialise currentMode to 0
             uint8_t currentMode = 0;
             status              = Attributes::CurrentMode::Get(endpointId, &currentMode);
+#ifdef EMBER_AF_PLUGIN_ON_OFF
+            // OnMode with Power Up
+            // If the On/Off feature is supported and the On/Off cluster attribute StartUpOnOff is present, with a
+            // value of On (turn on at power up), then the CurrentMode attribute SHALL be set to the OnMode attribute
+            // value when the server is supplied with power, except if the OnMode attribute is null.
+            if (emberAfContainsServer(endpointId, OnOff::Id) &&
+                emberAfContainsAttribute(endpointId, OnOff::Id, OnOff::Attributes::OnOff::Id, true) &&
```

Comment:
Could you please file a followup to get rid of the pointless last argument of emberAfContainsAttribute?